build.sbt

```diff
@@ -574,6 +574,19 @@ lazy val spark = (project in file("connectors/spark"))
     ),
     javaCheckstyleSettings("dev/checkstyle-config.xml"),
     Compile / compile / javacOptions ++= javacRelease11,
+    Test / compile / javacOptions := {
+      // lombok is only a dependency of test. So its path needs to be added explicitly.
```

Comment:
i dont get this. isnt lombok used in any other tests in any other module? 
if not, is there documentation what is needed for lombok to work with javac? this seems very esoteric for a library that just adds annotations.